src/styles/constants.scss

```diff
@@ -53,6 +53,29 @@ $xsmall: 14px;
 $xxsmall: 12px;
 $tiny: 10px;
 
+/* spacing */
```

Comment:
We need to discuss if this is the way we want to have spacing across the app.

Reply:
@mwogates @leviyehonatan In my opinion if we decide to go with this way it should be done in a separate PR dedicated to this so it will be consistent across the app.

Reply:
ok i will remove it in this PR

Reply:
it is true but since this part of the website is quite seperate maybe its ok to leave it, imo